easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.3.5-gompi-2023a.eb

```diff
@@ -0,0 +1,55 @@
+# Updated to v2.1.3 by
+# R.QIAO <rob.qiao@flinders.edu.au>
+# DeepThought, Flinders University
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.3.5'
+
+# HTTPS is not working
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'gompi', 'version': '2023a'}
+# Including 'usempi' will take precedence and override IQTREE_FLAGS and produces only 'iqtree-mpi' binary
+
+source_urls = ['https://github.com/iqtree/iqtree2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'IQ-TREE-2.3.5_use_EB_LSD2.patch',
```

Comment:
Is the [fix MPI patch](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.2.1_fix-mpi.patch) not required anymore?

Reply:
Probably not, otherwise building the MPI variant would fail, since that patch fixes compilation errors

Reply:
Don't know how/when that issue popped up before, but the same line is still there:
https://github.com/iqtree/iqtree2/blob/v2.3.5/pll/genericParallelization.h#L66

Reply:
If the build works, I don't see how the patch could be required. Maybe it was only required for older compiler/OpenMPI versions.